.github/workflows/build-test.yml

```diff
@@ -184,7 +184,7 @@ jobs:
         strategy:
             max-parallel: 1
             matrix:
-                image: [ 'nodejs-centos6-default', 'nodejs-centos7-node12', 'nodejs-centos7-fips' ]
```

Comment:
Please create a ticket to track nodejs-centos-fips upgrade node version and put it back for testing